test-utils/throttleMock.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function() {
+  var queue = []
+  return {
+    throttle: function(fn) {
```

Comment:
You mean having a top level `throttleMock()` factory? 

It allows to have a per-test queue. If a test forgets to clear its queue, it will not have effects on a subsequent test.

I've also added `afterEach` assertions to ensure that the queues are empty, but problems may arise when writing new tests/specs and the future author hasn't yet added such safeguards, it will not trigger errors in unrelated tests.